src/browsers/chromium/chromium.bzl

```diff
@@ -27,9 +27,9 @@ def define_chrome_repositories():
   
   browser_repo(
       name = "rules_browsers_chrome_mac_arm",
-      sha256 = "ea286d83bb28e4178dd376f208b85ed1ef325f731472dc32e4e3d44e9f366f5e",
-      # 135.0.7049.84
-      urls = ["https://storage.googleapis.com/chrome-for-testing-public/135.0.7049.84/mac-arm64/chrome-headless-shell-mac-arm64.zip"],
+      sha256 = "e8759d758b2e709bbe241d7b6828d8f531a115010bd476d13a3912967ce72336",
+      # 138.0.7204.49
```

Comment:
138 was the version that broke Components RBE.

Reply:
This is Chrome though. Should be fine?

Reply:
Ah nevermind. Didn't realize the Firefox and Chrome versions were so close 😅

Reply:
I was confused initially too